pkgs/by-name/wa/wayfreeze/package.nix

```
@@ -0,0 +1,33 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  libxkbcommon
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "wayfreeze";
+  version = "0-unstable-2024-05-23";
+
+  src = fetchFromGitHub {
+    owner = "Jappie3";
+    repo = pname;
```

Remove this pattern, see #277994
```suggestion
    repo = "wayfreeze";
```